### PR TITLE
Rename some resources and attributes

### PR DESCRIPTION
**Pull request by piscisaureus**

### CODE REVIEW

File: examples/provider-install-verification/main.tf [hunk 1]
```diff
@@ -58,7 +58,7 @@ resource "deno_domain_verification" "example_domain_verification" {
   }
 }
 
-resource "deno_certificate_provisioning" "example_certificate_provisioning" {
```
Comment (magurotuna) [inline examples/provider-install-verification/main.tf:61]: I see no change in the resource name in the resource definition. Is this change intended to be included in this PR?

File: examples/resources/deno_certificate_provisioning/resource.tf
Comment (magurotuna) [inline examples/resources/deno_certificate_provisioning/resource.tf:1]: Ditto (directory name needs to match the resource name so doc generator can use it)

**Review by magurotuna — CHANGES_REQUESTED** [review]

File: internal/provider/certificate_provisioning_resource.go
[the remark this answers is not in the document]
Comment (piscisaureus) [reply internal/provider/certificate_provisioning_resource.go:1]: good catch!

File: examples/resources/deno_certificate_provisioning/resource.tf
Comment (piscisaureus) [reply examples/resources/deno_certificate_provisioning/resource.tf:1]: ditto!

**Review by magurotuna — APPROVED** [review]
LGTM

**Pull request merged**